addons/website/views/snippets/s_masonry_block.xml

```diff
@@ -3,257 +3,234 @@
+            <h3><br/></h3>
+        </div>
+        <div class="o_grid_item o_grid_item_image g-height-10 g-col-lg-3 col-lg-3 text-center" data-name="Block" style="z-index: 2; grid-area: 1 / 2 / 11 / 5;">
             <img src="/web/image/website.s_masonry_block_default_image_2" class="img img-fluid mx-auto" alt=""/>
         </div>
-        <div class="o_grid_item col-lg-3 g-col-lg-3 g-height-5 text-center o_cc o_cc2" data-name="Block"
-            style="grid-area: 1 / 4 / 6 / 7; z-index: 2;">
-            <h3>A great title</h3>
-            <p>And a great subtitle</p>
+        <div class="o_grid_item o_cc o_cc3 g-height-5 g-col-lg-4 col-lg-4 justify-content-center text-center" data-name="Block" style="z-index: 3; grid-area: 1 / 5 / 6 / 9;">
+            <h3>Innovation <br class="d-none d-lg-inline"/>Hub</h3>
+            <p>Where ideas come to life</p>
         </div>
-        <div class="o_grid_item col-lg-3 g-col-lg-3 g-height-5 text-center o_cc o_cc4" data-name="Block"
-            style="grid-area: 6 / 4 / 11 / 7; z-index: 3;">
-            <h3>A great title</h3>
-            <p>And a great subtitle</p>
+        <div class="o_grid_item o_cc o_cc4 g-height-5 g-col-lg-4 col-lg-4 justify-content-center text-center" data-name="Block" style="z-index: 4; grid-area: 6 / 5 / 11 / 9;">
+            <h3>Community <br class="d-none d-lg-inline"/>Focus</h3>
+            <p>Building connections</p>
         </div>
-        <div class="o_grid_item col-lg-6 g-col-lg-6 g-height-10 o_grid_item_image text-center" data-name="Block"
-            style="grid-area: 1 / 7 / 11 / 13; z-index: 4;">
+        <div class="o_grid_item o_grid_item_image g-height-10 g-col-lg-3 col-lg-3 text-center" data-name="Block" style="z-index: 5; grid-area: 1 / 9 / 11 / 12;">
             <img src="/web/image/website.s_masonry_block_default_image_1" class="img img-fluid mx-auto" alt=""/>
         </div>
+        <div class="o_grid_item o_cc o_cc4 g-height-5 g-col-lg-1 col-lg-1 justify-content-center text-center d-none d-lg-flex o_snippet_mobile_invisible" data-name="Block" style="z-index: 6; grid-area: 6 / 12 / 11 / 13;">
+            <h3><br/></h3>
+        </div>
     </div>
 </template>
 
 <template id="s_masonry_block_mosaic_template" groups="base.group_user">
-    <div class="row o_grid_mode" data-row-count="13">
-        <div class="o_grid_item col-lg-3 g-col-lg-3 g-height-2 text-center o_cc o_cc3" data-name="Block"
-            style="grid-area: 1 / 1 / 4 / 4; z-index: 1;">
-            <h3>A great title</h3>
-            <p>And a great subtitle</p>
+    <div class="row o_grid_mode" data-row-count="10" style="gap: 16px;">
+        <div class="o_grid_item o_cc o_cc3 g-height-3 g-col-lg-3 col-lg-3 justify-content-start order-lg-0 rounded-3" data-name="Block" style="order: 1; z-index: 1; --grid-item-padding-y: 20px; --grid-item-padding-x: 20px; grid-area: 1 / 1 / 4 / 4; z-index: 1; linear-gradient(135deg, var(--o-color-4) -400%, var(--o-color-2) 100%);">
```

Comment:
The z-index is there twice, remove one of them :wink:

Reply:
Done 👍